README.md

```diff
@@ -60,6 +60,24 @@ cmake --build .
 
 You should now have an executable `create-data-mod` in the `build` folder. Hooray!
 
+### Debugging (GDB on Ubuntu)
+
+1. Ensure gcc tools are installed: `sudo apt-get install build-essential gdb`
+
+Now we have two debug options:
+
+2. Debug via GDB on the terminal...
+```
+bash ./buildForDebugger.sh
```

Comment:
You could also make `buildForDebugger.sh` executable (`chmod +x buildForDebugger.sh`) and directly execute `./buildForDebugger.sh` here, instead of executing it through `bash`.

Reply:
Ahhh OK, good suggestion.  But will that automatically be the case on clones, or will the user have to run that every time they clone the repo?  Or we could have an `init.sh` script that sets thing up and does that I suppose?

Reply:
Oh haha disregard, I will do this, apparently git has a way to keep track of this internally: https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git